cmd/collector/app/span_handler.go

```diff
@@ -125,6 +129,8 @@ func (h *zipkinSpanHandler) SubmitZipkinBatch(ctx thrift.Context, spans []*zipki
 		res.Ok = ok
 		responses[i] = res
 	}
+
+	h.logger.Debug("Zipkin span batch processed by the collector.", zap.Int("batch-size", len(responses)))
```

Comment:
nit: similar to above, s/batch-size/span-count/. Also, I'd use `len(spans)`, rather than indirect `len(responses)`.